src/[sandbox]/examples/webexWalkin.ts

```diff
@@ -0,0 +1,24 @@
+import "@/components/webexWalkin/WebexWalkin";
```

Comment:
So.  one thing that turned out to be a culpit in Momentum was absolute paths for types. Bringing in the component like this is fine( this line is fine) but if you were to try to only import the namespace per se, this will result in a type error. So just FYI, every time importing types, please use relative paths (@futurethang , @elenagiere)

Reply:
I remember doing the same when trying to resolve those typescript errors. Thanks for the reminder!